### PR TITLE
Use QualifiedName.getSuffix to get suffix

### PR DESCRIPTION
**Pull request by yaoxin226**

### CODE REVIEW

File: presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedNameReference.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public QualifiedName getName()
 
     public QualifiedName getSuffix()
     {
-        return QualifiedName.of(Iterables.getLast(name.getParts()));
```
Comment (yan-stein) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedNameReference.java]: @yaoxin226   Is that import ... Iterables  serving for this line only?

Comment (yaoxin226) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedNameReference.java]: @yan-stein Yes, nice catch. Removed this import. Thanks.

**Comment by cberner** [comment]
Merged. Thanks!

**Comment by yaoxin226** [comment]
Thanks @cberner for the review!

**Pull request closed without merging**